### ISSUE TITLE
Screencast does not render yahoo video and non-English text

### ISSUE THREAD

**Issue by rohrit** [chromedp/chromedp#1068]
I am using the page Screencast APIs to send the screencast PNGs to another browser to render. For some reason all videos on Yahoo show SS-501-300 error codes. Other sites like YouTube render and play fine. Another issue is non-English languages are not rendered correctly. Perhaps due to the version of chrome used in headless-shell.

#### What versions are you running?

```console
$ go list -m github.com/chromedp/chromedp
github.com/chromedp/chromedp v0.8.0
$ google-chrome --version
am using headless-shell:latest docker image
$ go version
go version go1.18.1 darwin/amd64
```

#### What did you do? Include clear steps.

```go
chromedp.ListenTarget(ctx, func(ev interface{}) {
		switch ev := ev.(type) {
		case *page.EventScreencastFrame:
			go func() {
				chrome := chromedp.FromContext(ctx)
				err := page.ScreencastFrameAck(ev.SessionID).Do(cdp.WithExecutor(ctx, chrome.Target))
				if err != nil {
					log.Printf("ScreencastFrameAck: %v", err)
					return
				}

				// Lock before write since websocket doesn't support concurrent writes
				conn.mu.Lock()
				err = conn.c.WriteMessage(websocket.TextMessage, []byte(ev.Data))
				conn.mu.Unlock()
				if err != nil {
					log.Printf("Error writing to websocket: %v", err)
					if strings.Contains(err.Error(), "close") {
						conn.stop()
					}
					return
				}
			}()

		}
	})

	// navigate
	if err := chromedp.Run(ctx,
		page.Enable(),
		page.StartScreencast().WithFormat(page.ScreencastFormatPng),
		chromedp.EmulateViewport(command.Width, command.Height),
		chromedp.Navigate(conn.url),
	); err != nil {
		log.Printf("could not execute screencast session: %v", err)
	}
```

#### What did you expect to see?
No errors and locales renders correctly

#### What did you see instead?
Y! video error
![Screenshot 2022-05-12 at 2 09 52 PM](https://user-images.githubusercontent.com/60731117/168031709-8a88c558-8187-4b93-a826-030a6520f45f.png)

non-English text issue
![Screenshot 2022-05-12 at 2 10 54 PM](https://user-images.githubusercontent.com/60731117/168031826-e07644a1-2f4c-4631-95af-2b973ddaf78b.png)

same page when viewed directly on Chrome - 
![Screenshot 2022-05-12 at 2 11 06 PM](https://user-images.githubusercontent.com/60731117/168031954-70028280-65b5-485a-a5fb-1fa89e44be5c.png)

**Comment by ZekeLu** [comment]
> non-English text issue

See #611, you need to install fonts for that language.

**Comment by rohrit** [comment]
Excellent! thanks for the tip. That fixed the language issue.

Please let me know if you know how the video issue can be fixed as well.

**Comment by ZekeLu** [comment]
I'm afraid that the video feature is not supported by `chromedp/headless-shell`. I have tested both youtube and yahoo in `chromedp/headless-shell`, neither works. I saw something like this in the browser log:

```
[0512/103503.678901:ERROR:web_contents_delegate.cc(227)] WebContentsDelegate::CheckMediaAccessPermission: Not supported.
```

When I tested in a local browser in headless mode, it works (both youtube and yahoo).

But you said "Other sites like YouTube render and play fine". That's weird, and I don't know why.

My suggestion is to try other docker image which ships full-feature chromium.
